blocks/library/gallery/index.js

```diff
@@ -121,16 +124,24 @@ registerBlockType( 'core/gallery', {
 			controls,
 			focus && images.length > 1 && (
```

Comment:
I think the description should be shown at all times, not just when there's more than 1 image. Of course, this check makes sense for the controls.